src/components/drawer/drawer.html

```diff
@@ -1,58 +1,45 @@
 <aside class="drawer mdc-drawer--open mdc-drawer--persistent mdc-typography">
-  <nav class="mdc-drawer__drawer">
+  <nav class="mdc-drawer__drawer" style="overflow: visible">
```

Comment:
do not use inline styles!!! here you can use more specific selector to make it work:

![image](https://user-images.githubusercontent.com/36405386/37422770-ac3912be-27c4-11e8-95a7-30022d52abc0.png)

![image](https://user-images.githubusercontent.com/36405386/37422781-b7a5e064-27c4-11e8-80b6-534a09c9246d.png)